doc/integrations/redshift.ipynb

```diff
@@ -0,0 +1,1182 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d22b2168-17c7-4a77-89d1-b928f3b7d41c",
+   "metadata": {},
+   "source": [
+    "# Redshift\n",
+    "\n",
+    "This tutorial will show you how to use JupySQL with Redshift.\n",
+    "\n",
+    "## Pre-requisites\n",
+    "\n",
+    "First, let's install the required packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "104f9ccb-1ce8-4850-a0f0-520cf445b292",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "%pip install jupysql sqlalchemy-redshift redshift-connector --quiet"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "06edf8cd-6c65-4018-a818-47037c3ae831",
+   "metadata": {},
+   "source": [
+    "Load JupySQL:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "ba102937-c7bd-48a4-aec0-999794519b02",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<span style=\"None\">Found pyproject.toml from &#x27;/Users/eduardo/dev/jupysql&#x27;</span>"
+      ],
+      "text/plain": [
+       "Found pyproject.toml from '/Users/eduardo/dev/jupysql'"
```

Comment:
Can we hide this cell output if not needed : `Found pyproject.toml from '/Users/eduardo/dev/jupysql'`